src/css/components/grid.scss

```diff
@@ -32,3 +32,77 @@
     }
   }
 }
+
+.col,
+.row {
+  position: relative;
+}
+
+.row {
+  align-items: center;
+  flex-wrap: wrap;
+
+  // hack for react containing div requirement
```

Comment:
these rules pass flex values on to unclassed child `div`s which are sometimes necessary in React templates